doc/sphinx-guides/source/installation/config.rst

```diff
@@ -1503,16 +1507,15 @@ dataverse.files.directory
 Please provide an absolute path to a directory backed by some mounted file system. This directory is used for a number
 of purposes:
 
-1. ``<dataverse.files.directory>/temp`` after uploading, data is temporarily stored here for ingest and/or before
+1. ``<dataverse.files.directory>/<PID Authority>/<PID Identifier>`` is the subdirectory layout when using the target
```

Comment:
dataverse.files.directory isn't used for permanent file storage for file stores - there are new store specific options that may point to the same place. DCM is probably legacy w.r.t. multiple stores and still uses this dir.

Reply:
Maybe something like this, and move it to the end since DCM is not uses/is deprecated?

 1. ``<dataverse.files.directory>/<PID Authority>/<PID Identifier>``  - The DCM feature for :doc:`../developers/big-data-support`
   is able to trigger imports for externally uploaded files from this area under certain conditions. (This directory may also be used by file stores for :ref:`permanent file storage <storage-files-dir>`, but this is controlled by other, store-specific settings.

Reply:
I integrated your suggestion into a new version. Please have another look. Thx!

Reply:
LGTM